### PR TITLE
Remove imagepullsecret after open sourcing

### LINKED ISSUE #30: Remove registryConfig from values.yaml
Since the image is public, we do not need the `registryConfig` variable in `values.yaml`

https://github.com/nirmata/kyverno-aws-adapter/blob/d6c32ab35b1680206c18a3c28c7e107eed4d6074/charts/kyverno-aws-adapter/values.yaml#L23-L27

### PR DESCRIPTION
**Pull request by pns-nirmata**
Since the repo is open source, we don't need imagepull secrets.

Fixes #30

### CODE REVIEW

**Comment by anusha94** [comment]
> Some users will mirror the image on a private repo though.

That's a good use case. Then can we add that appropriate comment for the field?

Something like "registry credentials if pulling from your private repository" ?

**Comment by eddycharly** [comment]
> > Some users will mirror the image on a private repo though.
> 
> That's a good use case. Then can we add that appropriate comment for the field?
> 
> Something like "registry credentials if pulling from your private repository" ?

We should introduce `imagePullSecrets` as a value, users won't put the credentials in clear text here. They will want to reference an existing secret in the cluster (bring your own image pull secret ;-)

**Comment by anusha94** [comment]
> bring your own image pull secret

@eddycharly Love it!

Let's open a new issue for it and address it separately. We can go ahead with removing this IMO.

**Comment by eddycharly** [comment]
Yeah, this is trivial to add.

**Comment by eddycharly** [comment]
@anusha94 see https://github.com/nirmata/kyverno-aws-adapter/pull/33

File: README.md [hunk 1]
```diff
@@ -8,29 +8,12 @@ You’ll need an [EKS](https://aws.amazon.com/eks/) cluster to run against.
 
 ### Running on the EKS cluster
 1. Make sure that you have configured an [IAM role for the service account](#IAM-Role-for-Service-Account) `kyverno-aws-adapter-sa` in your desired namespace (configured in `values.yaml`) and specified the role's ARN in the `roleArn` field inside `values.yaml` file.
```
Comment (shreemaan-abhishek) [inline README.md:10]: > specified the role's ARN in the `roleArn` field inside `values.yaml` file.

Where do I get this `values.yaml` file from? 🤔

Comment (shreemaan-abhishek) [inline README.md:10]: > specified the role's ARN in the `roleArn` field inside `values.yaml` file.

Where do I get this `values.yaml` file from? 🤔

**Comment by anusha94** [comment]
I think we can close this PR in favor of #33

File: README.md (hunk 1 above)
Comment (pns-nirmata) [reply README.md:10]: Yes. The documentation needs to be changed. To specify the roleArn at helm install time.

**Comment by pns-nirmata** [comment]
Closing this PR in favour of #33

**Pull request closed without merging**